cv-data-service-library/src/main/java/com/trihydro/library/helpers/TimGenerationHelper.java

```diff
@@ -1101,14 +1171,52 @@ private OdeGeoRegion getServiceRegion(List<Milepost> mileposts) {
         return serviceRegion;
     }
 
+    private Milepost attemptToCalculateAnchorPoint(TimUpdateModel tum,
+                                                   List<ResubmitTimException> exceptions,
```

Comment:
Issue (blocking): This list is passed by value, meaning any changes made to it will not persist in the caller's context. Instead of modifying the list directly, exceptions should be thrown within this method, and the caller should handle adding them to the exceptions list.

Reply:
Addressed